src/components/molecules/PageHeader.tsx

```diff
@@ -18,9 +19,59 @@ export default function PageHeader({
     center: center
   })
 
+  const [fontSize, setFontSize] = useState('')
+  const [lineHeight, setLineHeight] = useState('')
+
+  const setFontSizeAndLineHeight = (
+    fontSize: number,
+    linesNumber: number,
+    lineHeight: number
+  ) => {
+    if (linesNumber > 1) {
+      linesNumber += 4
+      fontSize -= linesNumber
+      lineHeight -= linesNumber
+      setFontSize(fontSize + 'px')
+      setLineHeight(lineHeight + 'px')
+    }
+  }
+
+  const countLines = (element: any) => {
+    const elementHeight = element.offsetHeight
+    const lineHeight = parseInt(
+      document.defaultView
+        .getComputedStyle(element, null)
+        .getPropertyValue('line-height')
+    )
+    const lines: number = elementHeight / lineHeight
+    return { linesNumber: lines, lineHeight: lineHeight }
+  }
+
+  useEffect(() => {
+    const titleElement = document.getElementsByClassName(
+      'PageHeader-module--title--2n3-e'
```

Comment:
this could go wrong since those classnames are auto-generated, we can only trust that `PageHeader-module--title` won't change (unless we rename the CSS module files or class names). Would suggest rather using CSS attribute selector via `querySelector` so you can do `class*='PageHeader-module--title'`